docs/about.rst

```diff
@@ -58,13 +58,17 @@ We think that's a pretty fair deal, and we're big believers in open source. If y
 Presentations
 -------------
 
-Yellowbrick has enjoyed the spotlight at a few conferences and in several presentations. We hope that these videos, talks, and slides will help you understand Yellowbrick a bit better.
+Yellowbrick has enjoyed the spotlight in several presentations at recent conferences. We hope that these notebooks, talks, and slides will help you understand Yellowbrick a bit better.
 
-Videos:
+Conference Presentations (videos):
```

Comment:
I changed the title to "Conference Presentations" since some of our contributors are working on short video tutorials that we will be able to add here.